backend/api/MQTT/MqttService.cs

```diff
@@ -105,11 +105,7 @@ private Task OnMessageReceived(MqttApplicationMessageReceivedEventArgs messageRe
                 return Task.CompletedTask;
             }
 
-            _logger.LogInformation(
-                "Topic: {topic} - Message recieved: \n{payload}",
-                topic,
-                content
-            );
+            _logger.LogDebug("Topic: {topic} - Message recieved: \n{payload}", topic, content);
```

Comment:
Are we sure about this? 
When running it in development it works for me, 
and I would think that changing `appsettings.Production.json` would only change logging for the production environment, or have I misunderstood something.
Ragarding information logging on database calls: yes it probably should be information level, however before we have implemented the timeseries database this will spam the log every second with a bunch of details. 
Getting rid of this spam was the main objective behind the issue, but we should probably make a blocked task to revert the level when the timeseries database is in place.

Reply:
Are we sure about this? 
When running it in development it works for me, 
and I would think that changing `appsettings.Production.json` would only change logging for the production environment, or have I misunderstood something.

Reply:
"This will only change the logging in production, not in development, maybe we should add this change to the appsettings.Development.json as well?"
Are we sure about this? 
When running it in development it works for me, 
and I would think that changing `appsettings.Production.json` would only change logging for the production environment, or have I misunderstood something.

Reply:
Oh, yes, true, the `appsettings.json` are the default values and will only be overwritten by values in `appsettings.Development.json` if the same values are specified there.
So this is a good approach as it will set the defaults and we can override in appsettings.Development if needed!
Nice job